ci/run_build.sh

```diff
@@ -39,6 +39,7 @@ sudo apt-get update
 
 TOP_DIR="$(pwd)"
 DEPS_DIR="${TOP_DIR}/deps"
+NUM_JOBS=$(nproc)
```

Comment:
just wondering if this won't freeze the system in some cases

Reply:
Hmm, you mean calling the `nproc` command? or using all cores for azure VMs which btw were already forced in some cases with hardcoded values (see for example: https://github.com/analogdevicesinc/no-OS/blob/bc77edd35f9f42a9fb5289e21994dca8fdad7bbe/ci/astyle.sh#L99) ?

Reply:
I mean using all cores. but if we already do it in some cases, I guess there is no issue